### PR TITLE
Popup Window

### PR DESCRIPTION
**Pull request by jlberbesi**
In this PR  I have done the following actions:

Implement the  interaction such as:

-When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
-When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi  @jlberbesi , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] Doesn't have any contribution from team members.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by didierganthier — COMMENTED** [review]
Hi Team,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project that doesn't include contributions from all team members while this milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#work-and-submission-mode).

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Comment by jlberbesi** [comment]
Update: My partner @Abdelaziz-Ali  already was invited to contribute to this branch.

**Comment by shella12** [comment]
> Update: My partner @Abdelaziz-Ali already was invited to contribute to this branch.

👍 Awesome now you may submit your PR for review from your dashboard

**Review by nahnahdev — COMMENTED** [review]
## Project Status: INVALIDATED :red_circle: 
Hi @jlberbesi , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that is missing the `Eslint linter checks`. SInce you are using Javascipt in this project, please ensure that you have properly installed the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in your project
- Your linter checks now
image
- After adding the javascript `Eslint` linters, you should have 5 linter checks in all :wink: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jlberbesi** [comment]
Screenshot 2023-03-11 at 2 31 10 PM

Eslint linter running without any problems. All bugs were fixed already.

**Comment by jlberbesi** [comment]
> ## Project Status: INVALIDATED 🔴
> Hi @jlberbesi ,
> 
> While you made a great effort in this project, I can't go ahead and review your code.  [ 68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f546955596457707454716e38794f6b3144702f67697068792e676966 ](https://camo.githubusercontent.com/b2f3c41adfb07c0b3409d41189048435adfe4d02ec7548dcac09378450e4e7ff/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f546955596457707454716e38794f6b3144702f67697068792e676966) [ ](https://camo.githubusercontent.com/b2f3c41adfb07c0b3409d41189048435adfe4d02ec7548dcac09378450e4e7ff/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f546955596457707454716e38794f6b3144702f67697068792e676966)
> 
> ## Invalid Code Review Request
> * This is because you have submitted a project, that is missing the `Eslint linter checks`. SInce you are using Javascipt in this project, please ensure that you have properly installed the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in your project
> * Your linter checks now
>   image
> * After adding the javascript `Eslint` linters, you should have 5 linter checks in all 😉
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @NahnahAJ in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Linterns fixed and all bugs were already fixed. No a problem anymore.

File: object.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      </div>
+    </div>
+  </article>
+`;
+
+const projectsContainer = document.getElementById('projects-container');
+const tonicContainer = document.createElement('div');
+tonicContainer.innerHTML = tonicHTML;
+
+projectsContainer.appendChild(tonicContainer);
```
Comment (Abenezer-Tilahun) [inline object.js:82]: - [ ] As stated in the project requirement Modal popup data Should be  populated with the data from the projects array.In the case of yours, your data is not Modal popup data is populated with the data from the projects array as shown below image.
- Your Popup window Look likes this : data is not populated from the array just like Figma design 
Capture

- The Figma Popup design Look likes this : 
Capture

- [ ] As shown in this [video](https://drive.google.com/file/d/1TWidr3nCCicS6OZOGZh_rwhMK-rWAV3Z/view?usp=share_link) the popups for both mobile and desktop screen sizes doesn't work as expected it only popup for one of the project list among the four list , So kindly Fix it.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @jlberbesi , 

Great job for implemented  the Project requirement  , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git Hub -Flow.
✔️ Good commit messages
✔️ Professional READEME 


almost

## Required Changes ♻️
_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: object.js (hunk 1 above)
Comment (jlberbesi) [reply object.js:82]: @Abdelaziz-Ali  Alright. I'll change it then. 

Thank you for your review.

**Pull request merged**